cmd/commands.go

```diff
@@ -242,6 +242,19 @@ provisioning stages.`,
 		}(),
 	})
 
+	RegisterCommand(Command{
+		Name: "format",
```

Comment:
Let's call it `fmt` as an homage to `go fmt` :)

Reply:
I actually named it `format` since all the other command names were expressive and this seemed like an exception among them. But I do like `fmt` too.

Reply:
Gotcha. I'm torn. I don't care either way :)